fmriprep/cli/parser.py

```diff
@@ -570,11 +564,23 @@ def _bids_filter(value):
 
 def parse_args(args=None, namespace=None):
     """Parse args and run further checks on the command line."""
+    import os
     import logging
     from niworkflows.utils.spaces import Reference, SpatialReferences
 
     parser = _build_parser()
     opts = parser.parse_args(args, namespace)
+
+    new_config = os.getenv('FMRIPREP_NO_REUSE_CONFIG')
+    if not new_config:
+        # attempt to reuse previous config
+        config.load_previous_config(
+            opts.output_dir,
+            opts.work_dir,
+            participant=opts.participant_label,
+            new_uuid=not opts.reports_only,
+        )
```

Comment:
This should not be a default behavior.

Reply:
I tend to agree here, but I think @oesteban felt otherwise?